miio/yeelight.py

```diff
@@ -85,6 +85,109 @@ def name(self) -> str:
         """Return the internal name of the bulb."""
         return self.data["name"]
 
+    @property
+    def color_flowing(self) -> bool:
+        """Return whether the color flowing is active."""
+        return bool(int(self.data["flowing"]))
+
+    @property
+    def color_flow_params(self) -> Optional[str]:
+        """Return color flowing params."""
+        if self.color_flowing:
+            return self.data["flow_params"]
+        return None
+
+    @property
+    def delay_off(self) -> int:
+        """Return delay in minute before bulb is off."""
+        return self.data["delayoff"]
```

Comment:
I think all data is returned as strings, so this (and maybe others) will require casting. Could you please check that?

Reply:
I did not notice, fixed in e06a580. Thanks.